frontend/src/components/pages/Account.tsx

```diff
@@ -75,9 +75,11 @@ const Account = (): JSX.Element => {
         );
       }
       setDonor(donorResponse);
-      setBusinessName(donorResponse.businessName);
+      if (authenticatedUser?.role !== Role.VOLUNTEER)
+        setBusinessName(donorResponse.businessName);
+      else setBusinessName("");
     };
-    getDonor();
+    if (authenticatedUser?.role !== Role.VOLUNTEER) getDonor();
```

Comment:
can we lift the `getDonor` function outside of the `useEffect`?

Reply:
also maybe rename to something more reflective of the function now that we have multiple roles?